src/Codeception/Lib/Driver/Db.php

```diff
@@ -221,7 +221,14 @@ public function executeQuery($query, array $params)
             throw new \Exception("Query '$query' can't be prepared.");
         }
 
-        $sth->execute($params);
+        $i = 0;
+        foreach ($params as $value) {
+            $i++;
+            $type = preg_match('/^[-+]?\d+$/', $value) ? \PDO::PARAM_INT : \PDO::PARAM_STR;
```

Comment:
Determining whether the column and/or value represent booleans is near impossible whereas an integer check is entirely feasible and allows `0`/`1` to be passed for boolean values. Can implement logic here to enable passing `true`/`false` values as well.